app/src/main/res/layout/main_activity.xml

```diff
@@ -113,6 +113,16 @@
                     app:cardUseCompatPadding="true"
                     app:layout_constraintTop_toBottomOf="@id/sub_text"
                     app:shapeAppearance="@style/ShapeAppearance.Material3.Corner.ExtraLarge" />
+
+                <TextView
+                    android:id="@+id/text_title"
```

Comment:
Give a more meaninful name to this text view, since it's only used with the "No ROMs found" text.